test.py

```diff
@@ -34,6 +34,7 @@
 from io import StringIO
 from scripts import coverage    # type: ignore
 from test.pylib.artifact_registry import ArtifactRegistry
+from test.pylib.cpp.common_cpp_conftest import ninja, get_configured_modes
```

Comment:
Can you move these functions to `test.pylib.util` or some new file under `test.pylib`?